### PR TITLE
Update documentation for Jade recipe

### PR DESCRIPTION
**Pull request by 808loud**
Please forgive me if I'm not following protocol with this PR... this is my first contribution to a public project :octocat:

### CODE REVIEW

**Comment by silvenon** [comment]
Congratulations for your first contribution! Sorry for getting around to your PR so late 😅 Everything is great except one tiny indentation bit in the following comment…

File: docs/recipes/jade.md [hunk 1]
```diff
@@ -111,7 +112,8 @@ Wiredep supports Jade:
        exclude: ['bootstrap-sass'],
        ignorePath: /^(\.\.\/)*\.\./
      }))
-     .pipe(gulp.dest('app/layouts'));
+-     .pipe(gulp.dest('app'));
++     .pipe(gulp.dest('app/layouts'));
```
Comment (silvenon) [inline docs/recipes/jade.md]: Both of these lines have one extra space in the indentation.

Comment (808loud) [inline docs/recipes/jade.md]: Weird... it looks perfectly aligned in the Preview pane. I will fix it now.

**Comment by 808loud** [comment]
Weird... it looked perfectly aligned in the Preview pane. I will fix it now.

**Comment by 808loud** [comment]
~~Fixed. Looks like when marking up a diff, the editor requires one less space in a line starting with a "+" as compared to a line starting with a "-".~~

~~I had never noticed that before. I'm guessing that I had (mistakenly) manually altered it to make it line up in the editor.~~

😂 🍻

**Comment by silvenon** [comment]
I was talking about the two lines right above my comment, not the ones you edited now 😁 Could you revert that change and fix those instead?

**Comment by 808loud** [comment]
Derp. Will fix now.

**Comment by silvenon** [comment]
Great! I squashed your commits and pushed them manually, because squashing can be a drag for new contributors. Congratulations for your first contribution! 🎉

**Pull request closed without merging**